lib/src/main/kotlin/nodes/abstractNode.kt

```diff
@@ -0,0 +1,6 @@
+package nodes
+
+abstract class abstractNode<K: Comparable<K>, V, someNode: abstractNode<K, V, someNode>>(val key:K, var value: V) {
```

Comment:
Ссылаясь на тот же [CC](https://kotlinlang.org/docs/coding-conventions.html#colon): тут немного не хватает пробелов.
```suggestion
abstract class abstractNode<K : Comparable<K>, V, someNode : abstractNode<K, V, someNode>>(val key: K, var value: V) {
```